aws/cloudformation-templates/base/_template.yaml

```diff
@@ -2,12 +2,12 @@
 AWSTemplateFormatVersion: 2010-09-09
 
 Description: >
-    This template deploys the Retail Demo Store.
+  This template deploys the Retail Demo Store.
 
 Conditions:
   CreateElasticsearchServiceLinkedRole: !Equals
     - !Ref CreateElasticsearchServiceLinkedRole
-    - 'Yes'
+    - "Yes"
```

Comment:
I suppose all these single to double quotes changes come from cfn-lint. which is good.
Was it ran on every cfn template or just the one impacted by the PR?

It makes a big PR, hard to decipher from the real changes.

Reply:
Yes, sorry it came from the Linter in my editor. Only run on those impacted by the PR during a save event.